src/Approximations/overapproximate.jl

```diff
@@ -516,3 +516,180 @@ function overapproximate(cap::Intersection{N,
     return overapproximate(swap(cap), dir; kwargs...)
 end
 
+
+function load_taylormodels_overapproximation()  # function to be loaded by Requires
+return quote
+"""
+         overapproximate(vTM::Vector{TaylorModel1{T, S}},
+                            ::Type{Zonotope})where{T, S}
+
+Convert between TaylorModel1 and in Zonotope.
+
+### Input
+
+
+- `vTM` -- Vector of TaylorModel1
+- `[P]` -- target type
+
+### Output
+
+The Zonotope converted from vector of TaylorModel1 as the target set.
+
+### example
+
+julia> using TaylorModels,IntervalArithmetic,Revise
```

Comment:
```suggestion
```julia
julia> using TaylorModels, IntervalArithmetic

julia> I = Interval(-0.5, 0.5);

julia> x₀ = Interval(0.0);

julia> D = Interval(-3.0, 1.0);

julia> p = Taylor1([2.0, 1.0], 2)
 2.0 + 1.0 t + 𝒪(t³)

julia> q = Taylor1([0.9, 3.0], 2)
 0.9 + 3.0 t + 𝒪(t³)
 
 julia> TM = [TaylorModel1(p, I, x₀, D),TaylorModel1(q, I, x₀, D)]
2-element Array{TaylorModel1{Float64,Float64},1}:
 2.0 + 1.0 t + [-0.5, 0.5]
 0.9 + 3.0 t + [-0.5, 0.5]
 
 julia> Z = overapproximate(Zonotope,TM)
Zonotope{Float64}([1.0, -2.1],
   [1, 1]  =  2.0
   [2, 1]  =  6.0
   [1, 2]  =  0.5
   [2, 3]  =  0.5)
```

- For code examples use triple backticks and `julia`. If the code is meant to be a doctest, use `jldoctest`. In this example i think it is fine without doctest.
- No need for `Revise`.
- The variable `m` is not used! We can remove it.

Reply:
1>  I am going to add tests soon.
2>  Yes Algorithm briefing is needed, will include them in next commit.

Thank you!